rabbitmq/metadata.csv

```diff
@@ -29,6 +29,7 @@ rabbitmq.queue.active_consumers,gauge,,,,"Number of active consumers, consumers
 rabbitmq.queue.bindings.count,gauge,,,,Number of bindings for a specific queue,0,rabbitmq,queue bind
 rabbitmq.queue.consumers,gauge,,,,Number of consumers,0,rabbitmq,cons
 rabbitmq.queue.consumer_utilisation,gauge,,fraction,,The ratio of time that a queue's consumers can take new messages,0,rabbitmq,cons util
+rabbitmq.queue.head_message_timestamp,gauge,,,,Timestamp of the head message of the queue,0,rabbitmq,head msg ts
```

Comment:
Could you indicate the unit of the timestamp?

Reply:
As I said, we can't be sure of the unit of the timestamp since it can be chosen by the user. If it uses this plugin https://github.com/rabbitmq/rabbitmq-message-timestamp it is in `ms` for sure, but not everyone use it.
Should we mark in as `ms` anyway ?

Reply:
I think so. Users can establish overrides on their datadog account